pkg/repo/fs.go

```diff
@@ -209,22 +197,44 @@ func (fsr *FsRepo) WriteRunInfo(ctx context.Context, summaryShellVariablesString
 }
 
 // EnsureRepoPathsConfigured modifies the config to include keys for accessing repo paths
-func (fsr *FsRepo) EnsureRepoPathsConfigured(c config.ReadWriter) {
-	c.SetIfAbsent(types.AuthTokensPath, fsr.join(config.TokensPath))
-	c.SetIfAbsent(types.UserKeyPath, fsr.join(config.UserPrivateKeyFileName))
-	c.SetIfAbsent(types.NodeExecutorPluginPath, fsr.join(config.PluginsPath))
+func (fsr *FsRepo) EnsureRepoPathsConfigured(c config_legacy.ReadWriter) {
+	c.SetIfAbsent(legacy_types.AuthTokensPath, fsr.join(config_legacy.TokensPath))
```

Comment:
you are still creating compute directory when running a requester, and vice versa

Reply:
This method is only called in the `v2_3.go` and has bee left unmodified to maintain the behavior of that migration.